### DISCUSSION TITLE
Commander PRO Temp cycles to 0 then to temp repeatedly

### DISCUSSION THREAD

**Discussion by JDLSuper55** [EvanMulawski/FanControl.CorsairLink#177] [Q&A]
I'm having an issue and its seems to be random. I have a custom loop so I'm controlling my pump speed via the water temp. But there are times my water temp will randomly drop to 0 and then to correct temp and repeat. Which is causing the pump and fans to surge. Any ideas?

https://github.com/EvanMulawski/FanControl.CorsairLink/assets/174540889/2046ccc7-462d-44cc-ab15-e91329a229a1

**Comment by EvanMulawski** [comment]
Hi @JDLSuper55, does HWiNFO64 show the same thing? This looks like an incompatible program is running and communicating with the device.

**Reply by JDLSuper55** [reply]
Yeah HWiNFO64 does show it drop to zero. It just seems to do it at random times and only last for a second. I ended up changing the response time on my curve to 5 seconds so it doesn't keep cycling the fans and pump. Would still like to figure out how to fix it.

**Reply by JDLSuper55** [reply]
I think I figured it out. Seems OPENRGB is causing the issue, as soon as I close it, it doesn't drop down to 0 anymore. Any idea on to fix it?

**Reply by EvanMulawski** [reply]
Ah, there should be a compatibility toggle in the SignalRGB device settings for the Commander Pro. That should be turned on.

**Reply by EvanMulawski** [reply]
For OpenRGB, v1.0 is compatible but has not been released yet. You can use this build until the official release is available: https://github.com/EvanMulawski/FanControl.CorsairLink/issues/101#issuecomment-1806536056

**Comment by DarkMiata** [comment]
Hello, i have the same problem when Icue is launch in the same time than FanControl
